src/main/java/org/altbeacon/beacon/distance/ModelSpecificDistanceCalculator.java

```diff
@@ -245,6 +259,19 @@ else if (code != 200) {
         }).executeOnExecutor(AsyncTask.THREAD_POOL_EXECUTOR);
     }
 
+    void _buildModelMapTest(String jsonString) throws JSONException{
```

Comment:
See my other comment above about removing the private modifier from the main method (leaving it with no modifier or changing to protected) .